packages/@tailwindcss-upgrade/src/codemods/migrate-tailwind-directives.test.ts

```diff
@@ -412,7 +414,7 @@ it('should replace `@responsive` with its children', async () => {
     `),
   ).toMatchInlineSnapshot(`
     ".foo {
-      color: red;
-    }"
+        color: red;
+      }"
```

Comment:
This is formatted kinda weird isn't it?

Reply:
Ah yep, I'll see if I can clean this up. What's happening is that:

1. This is part of the "user" CSS bucket, which we don't format
2. The `@responsive` just replaces the outer layer with its children. This results in CSS like this:

```css
··.format·{
····color:·red;
··}
```
Added `·` to show the spaces, but that's why it's being formatted this way.

Reply:
Solved it by marking each node inside `@responsive` with `tailwind_pretty`